src/main/java/org/opensearch/security/securityconf/impl/v7/ConfigV7.java

```diff
@@ -123,6 +120,7 @@ public static class Dynamic {
         public String hosts_resolver_mode = "ip-only";
         public String transport_userrname_attribute;
         public boolean do_not_fail_on_forbidden_empty;
+        public Extensions extensions = new Extensions();
     
         @Override
         public String toString() {
```

Comment:
Can you update this `toString` and also add `toString()` to the `Extensions` class similar to how `Kibana` is done?

Reply:
Is there a use case for allowing this part of the configuration to be printed? Since it stores encryption/signing keys I think it would be a good idea to hide it as much as possible.

Reply:
So it's ok to include this secret strings in toString() methods. Pushed my changes